.github/ISSUE_TEMPLATE/feature-change.md

```diff
@@ -0,0 +1,65 @@
+---
+name: Feature/Change
+about: Toevoegingen en wijzigingen
+title: "COMPONENT: SAMENVATTING"
+labels: ""
```

Comment:
Waarom geen labels? Is dat omdat deze ISSUE_TEMPLATE voor 2 types gebruikt wordt? Is het verplicht om `++labels: ""` te vermelden? Of zou het weg kunnen?

Op https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms lees ik dat het niet verplicht is en ook 
```
Labels that will automatically be added to issues created with this template. If a label does not already exist in the repository, it will not be automatically added to the issue.
```

Is dit dan niet een argument om separate `ISSUE_TEMPLATE`s te hanteren voor `Feature` en `Change`, zodat `labels` gevuld kan worden en tijdens aanmaken van het issue het label-veld meteen automatisch gevuld gaat worden?

Reply:
Mijn voorstel is dus om deze te `ISSUE_TEMPLATE` te splitsen in een `feature.md` en een `change.md`

Reply:
Prima. Dan zijn we ook beter voorbereid op wijzigingen aan deze templates.